main-settings/src/main/scala/sbt/Reference.scala

```diff
@@ -94,13 +94,4 @@ object Reference {
     case BuildRef(b)      => Some(b)
     case _                => None
   }
-
-  @deprecated("Explicitly wrap the URI in a call to RootProject.", "0.13.0")
-  implicit def uriToRef(u: URI): ProjectReference = RootProject(u)
```

Comment:
I think 4 years is sufficient time to allow users to migrate before we drop the maintenance cost.. https://github.com/sbt/sbt/commit/0507a2a1a9be5006b6ffb4dee3318796b70fe290